### PR TITLE
Encourage use of humans.txt

### PR DESCRIPTION
**Pull request by pmclanahan**
Just thought it was a good idea.

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,3 @@
+See root_files/humans.txt
+
+Please add yourself when you contribute code if you like.
```
Comment (schalkneethling) [inline AUTHORS]: Should we not perhaps rephrase this to be something like:

    "Software is made by humans, so if you have contributed to Bedrock, please add your Github username to the
      above mentioned [humans.txt][] file. Thank you for your contribution.

    [humans.txt]: https://github.com/mozilla/bedrock/blob/master/root_files/humans.txt"

**Comment by schalkneethling** [comment]
Other than a small suggestion, Yes, \o/ r+

File: AUTHORS (hunk 1 above)
Comment (pmclanahan) [inline AUTHORS]: this isn't a markdown file. Could make it one I guess. Figured it would just be txt.
